Prox/Prox/Models/AppState.swift

```diff
@@ -6,19 +6,37 @@ import Foundation
 
 class AppState {
     enum State: String {
-        case loading, details, carousel, background, exiting, unknown
+        case initial, loading, permissions, details, carousel, background, unknown
     }
 
     private static var cardVisits = Set<Int>()
 
     // Initial app state
-    private static var state = State.loading
+    private static var state = State.initial
     private static var preBackgroundState: State?
 
     static func getState() -> State {
```

Comment:
This sounded a little complicated, so I passed on doing this right now, but I'll look into it next.